docs: persist testing (#398)
rune/rune: docs/docs/advanced/persisted-data.md

The [Dev UI](../playtesting/simulating-multiplayer.md) provides a way to see and manipulate `game.persisted` so you can test that your game works across game sessions as intended. For instance, you can set one of the players' levels to 99 and check that it works fine when another new player joins as level 1.   

> If you decide to test persistence using the app, be aware that in review/draft game versions behave differently to published ones.

Reviewer: ```suggestion
If you decide to test persistence using the app, be aware that review/draft game versions behave differently to published ones.
```

> If you decide to test persistence using the app, be aware that in review/draft game versions behave differently to published ones.

Reviewer 2: ```suggestion
It's a good idea to test your in game in the app as well. If so, be aware that review/draft game versions behave differently to published ones.
```

Nice to encourage testing in the app :)

The game will be loaded with persisted data from the last available active game version (if there is any), but no data will be saved.

> This will guarantee that even if in review version has issues, it will not impact your experience in the published version.

Reviewer: ```suggestion
This guarantees that even if in review version has issues, it will not impact your experience in the published version.
```
